PupDates-CrossPol/pupdates_frontend
# Sk refactor home

#### What's this PR do?

- Centers the content of the login form and forces the text input to lower case (yay!)

- Selects a random user upon login for the logged-in user to view and eventually swipe on

- Updates the URL for the getAllUsers apiCall

#### Where should the reviewer start?

- In the LoginForm component, then apiCalls, then SwipeDogCard.js

#### How should this be manually tested?

Login, make sure once you login you are taking to the Home component which should show you a user that is not yourself

#### Screenshots (if appropriate)

<img width="363" alt="Screen Shot 2020-01-06 at 3 17 22 PM" src="https://user-images.githubusercontent.com/46407593/71853111-b26e7600-3097-11ea-93f5-84690cc24542.png">
